docs/content/direct/contributor.md

```diff
@@ -15,7 +15,7 @@ make test
 
 ## Integration testing
 
-There are currently two integration tests. Contributors can run them. There is also [a GitHub workflow](../../../.github/workflows/pr-test-integration.yml) that runs these tests.
+There are currently two integration tests. Contributors can run them. There is also a GitHub workflow (in `.github/workflows/pr-test-integration.yml`) that runs these tests.

```

Comment:
I think the formal/complete name is GitHub 'Actions' workflow.

Reply:
Fixed, thanks.